src/Compilers/CSharp/Portable/Compiler/MethodCompiler.cs

```diff
@@ -1057,6 +1055,9 @@ public override object VisitField(FieldSymbol symbol, TypeCompilationState argum
                                     analyzedInitializers,
                                     previousSubmissionFields,
                                     compilationState,
+                                    false,
```

Comment:
We might well wish to incorporate instrumentation of these initializers, but that change isn't necessary in order to achieve parity with the previous implementation.